app/src/main/java/com/mithridat/nonoconverter/ui/editimage/FragmentMain.java

```diff
@@ -26,20 +25,20 @@ public void onCreate(Bundle savedInstanceState) {
     @Override
     public View onCreateView(LayoutInflater inflater, ViewGroup container,
                              Bundle savedInstanceState) {
-        View _viewMainFragment = inflater.inflate(R.layout.fragment_main, null);
+        View viewMainFragment = inflater.inflate(R.layout.fragment_main, null);
 
-        _viewMainFragment
+        viewMainFragment
                 .findViewById(R.id.button_crop)
                 .setOnClickListener((View.OnClickListener) getActivity());
-        _viewMainFragment
+        viewMainFragment
```

Comment:
Please check [Method chain case](https://github.com/nonoteam/NonoConverter/blob/master/CodingStandard.md#line-wrapping-strategies) in CodingStandard.md